.project/spec.yaml

```diff
@@ -0,0 +1,100 @@
+specVersion: v2
+specMinorVersion: 2
+meta:
+  name: generative-virtual-screening
+  image: generative-virtual-screening
+  description: ""
+  labels: []
+  createdOn: "2024-10-06T16:45:44Z"
+  defaultBranch: main
+layout:
+- path: src/
+  type: code
+  storage: git
+- path: deploy/
+  type: code
+  storage: git
+environment:
+  base:
+    registry: nvcr.io
+    image: nvidia/ai-workbench/python-basic:1.0.2
+    build_timestamp: "20231114175838"
+    name: Python Basic
+    supported_architectures: []
+    cuda_version: ""
+    description: A Python Base with Jupyterlab
+    entrypoint_script: ""
+    labels:
+    - ubuntu
+    - python3
+    - jupyterlab
+    apps:
+    - name: jupyterlab
+      type: jupyterlab
+      class: webapp
+      start_command: jupyter lab --allow-root --port 8888 --ip 0.0.0.0 --no-browser
+        --NotebookApp.base_url=\$PROXY_PREFIX --NotebookApp.default_url=/lab --NotebookApp.allow_origin='*'
+      health_check_command: '[ \$(echo url=\$(jupyter lab list | head -n 2 | tail
+        -n 1 | cut -f1 -d'' '' | grep -v ''Currently'' | sed "s@/?@/lab?@g") | curl
+        -o /dev/null -s -w ''%{http_code}'' --config -) == ''200'' ]'
+      stop_command: jupyter lab stop 8888
+      user_msg: ""
+      logfile_path: ""
+      timeout_seconds: 60
+      icon_url: ""
+      webapp_options:
+        autolaunch: true
+        port: "8888"
+        proxy:
+          trim_prefix: false
+        url_command: jupyter lab list | head -n 2 | tail -n 1 | cut -f1 -d' ' | grep
+          -v 'Currently'
+    programming_languages:
+    - python3
+    icon_url: ""
+    image_version: 1.0.2
+    os: linux
+    os_distro: ubuntu
+    os_distro_release: "22.04"
+    schema_version: v2
+    user_info:
+      uid: ""
+      gid: ""
+      username: ""
+    package_managers:
+    - name: apt
+      binary_path: /usr/bin/apt
+      installed_packages:
+      - curl
+      - git
+      - git-lfs
+      - python3
+      - gcc
+      - python3-dev
+      - python3-pip
+      - vim
+    - name: pip
+      binary_path: /usr/local/bin/pip
+      installed_packages:
```

Comment:
If I remember right, the notebook also makes use of the third-party `requests` module. This could be a place to consider installing it. Not required as it is useful to point the user to how to install it, but it should speed up setup of the container.

Reply:
Good call out. We can make sure requests is installed in the project container.

Reply:
Will add requests as a pip requirement